index.d.ts

```diff
@@ -0,0 +1,214 @@
+import * as React from 'react';
```

Comment:
Concatenating a bunch of these doesn't work because you have duplicate imports. Just dumping this into a single file. We just need to re-write these in TS.